source/wp-content/themes/wporg-developer-2023/js/function-reference.js

```diff
@@ -34,8 +34,8 @@ jQuery( function ( $ ) {
 
 		$element.wrap( '<div class="wporg-developer-code-block"></div>' );
 
-		const $copyButtonContainer = $( '<div class="wp-block-button is-style-outline"></div>' );
-		const $copyButton = $( '<a href="#" class="wp-block-button__link wp-element-button"></a>' );
+		const $copyButtonContainer = $( '<div class="wp-block-button is-style-outline is-small"></div>' );
+		const $copyButton = $( '<a href="#" class="wp-block-button__link wp-element-button has-background"></a>' );
```

Comment:
Without this the background was being overwritten

![Screenshot 2023-11-07 at 4 02 26 PM](https://github.com/WordPress/wporg-developer/assets/1017872/4ebcc959-5df1-4a09-964b-82d6567662cd)